Dragonflight/WarriorArms.lua

```diff
@@ -231,13 +235,13 @@ spec:RegisterAuras( {
     },
     colossus_smash = {
         id = 208086,
-        duration = function () return 10 + (talent.blunt_instruments.enabled and 3 or 0 ) end,
+        duration = function () return 13 + ( talent.blunt_instruments.enabled and 3 or 0 ) end,
```

Comment:
As far as I can tell, Colossus Smash's baseline duration is 10 and is increased to 13 with Blunt Instruments.  Where is the 13s baseline duration from?  Is it the set bonus (which should be turned off in prepatch)?

Reply:
Fixed typo back to 10 base, and 13 with blunt instruments in https://github.com/Hekili/hekili/pull/1670/commits/ebf54a6f30823c1b686e2c89ad2cb4bff9f69a6d